build/source/dshare/get_ixname.f90

```diff
@@ -21,6 +21,7 @@
 module get_ixname_module
 ! used to get the index of a named variable
 USE nrtype                                          ! variable types, etc.
+USE multiconst,only:integerMissing
```

Comment:
No need to do anything here for this PR. But just to note: We currently have `integerMissing` in both `globalData.f90` and `multiconst.f90`.

See issue #140